scripts/build/transform/index.js

```diff
@@ -8,7 +8,7 @@ import { SOURCE_DIR } from "../../utils/index.mjs";
 const generate = babelGenerator.default;
 const atHelperPath = fileURLToPath(new URL("../shims/at.js", import.meta.url));
 
-/* Doesn't work for dependencies, optional chaining, and spread arguments */
+/* Doesn't work for dependencies, optional call, computed property, and spread arguments */
 
 // `Object.hasOwn(foo, "bar")` -> `Object.prototype.hasOwnProperty.call(foo, "bar")`
 function transformObjectHasOwnCall(node) {
```

Comment:
A handy little trick you might like.

![image](https://user-images.githubusercontent.com/30521560/192191995-c3b5130f-5eb0-476c-8187-5c15d0800bf1.png)

Reply:
Thank you! Looks good.